api-module-library/sharepoint/defaultConfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "microsoft-sharepoint",
+    "name": "microsoft-sharepoint-v2",
```

Comment:
@roboli I think we keep this as normal here and override it locally in the Frigg App. Anyone coming to this fresh shouldn't have to wonder what "v2" is all about. You'd override it in the "entities" directory, and either change the config variable or the GetName method.